grpc/service_config/service_config.proto

```diff
@@ -242,12 +255,13 @@ message LoadBalancingConfig {
     // xDS-based load balancing.
     // The policy is known as xds_experimental while it is under development.
     // It will be renamed to xds once it is ready for public use.
+    CdsConfig cds = 6;
```

Comment:
Shouldn't this have a json annotation of the form `[json_name="experimental_cds"]`